documentation/source/NuRadioReco/pages/times.rst

```diff
@@ -1,52 +1,95 @@
+method for `Trigger <NuRadioReco.framework.trigger.Trigger>` objects.
 
-We smoothly filter the first 5% and last 5% of the trace using a Tukey window function. This is a function that goes smoothly from 0 to 1.
-To avoid rollover of the pulse, we add 128ns of zeros to the beginning and end of the trace. Steps 2) and 3) are performed by the channelStopFilter module
-Both electric fields and channels have a trace_start_time variable. The get_times() function will automatically add the trace_start_time to the time array.
+Time delays are introduced by several hardware components. These time delays are often accounted for by folding/unfolding the complex transfer function (for an amplifier e.g. the measurement of the S12 parameter).
+The unfolding is typically done in the frequency domain where a convolution becomes a simple multiplication.
+As a consequence of typically short trace length (<~1000 samples) and because a Fourier transform implies implicitly a periodic signal,
+a pulse being at the beginning of the trace can end up being at the end of the trace.
+This can be avoided by using the `NuRadioReco.modules.channelStopFilter` module, which appends zeros at either end of the trace
+and applies a Tukey window to taper the ends of the trace towards zero.
 
-ARIANNA specific details:
+.. Note::
+  For the **ARIANNA** experiment, the hardware produces an artifact (a glitch) at the STOP position (i.e. the physical beginning of the trace).
+  Because of the way the hardware works, the STOP position is not at the beginning of the trace but can be anywhere.
+  During read in of the snowshovel calibrated data files, the trace is rolled such that the physical beginning (the STOP position) is at sample zero of the trace.
+  This glitch is removed by the `channelStopFilter <NuRadioReco.modules.channelStopFilter>` procedure described in the :ref:`module overview below <NuRadioReco/pages/times:Overview of modules that affect time>` .
+
+Station time (Event time)
 -------------------------
-Our hardware produces an artifact (a glitch) at the STOP position (i.e. the physical beginning of the trace). Because of the way the hardware works, the STOP position is not at the beginning of the trace but can be anywhere. During read in of the snowshovel calibrated data files, the trace is rolled such that the physical beginning (the STOP position) is at sample zero of the trace. This glitch is removed by the procedure described above.
+The global time at which the event takes place is stored as the `event time <NuRadioReco.framework.event.Event.get_event_time>`
+in the `Event <NuRadioReco.framework.event.Event>` object.
+This time usually corrsponds to the "vertex time" of the first interaction for simulations,
+and the time at which the data was recorded in the DAQ for data.
+It is stored as an `astropy.time.Time` object to enable sub-ns precision on the absolute time.
```

Comment:
Hm that is only true for data, right? In simulations this is typically also "just" a float. Should we change that?